scrc/data_classes/ruling_citation.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, citation_str, language):
             raise ValueError(f"The Citation String ({citation_str}) could not be parsed successfully.")
 
     def __str__(self):
-        return f"{self.ruling_str} {self.year} {self.volume} {self.page_number}"
+        return f"{self.ruling_str}-{self.year}-{self.volume}-{self.page_number}"
```

Comment:
What is the reason for this?
I think usually it is without dashes: e.g. BGE 132 II 257

Reply:
The reason is that I want to compare this string to given bge file_number. Those file_numbers are stored in our db with dashes.